.releaserc.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
     branches: [
         {name: 'main'},
         {name: '1.x.x', range: '1.x.x', channel: '1.x.x'},
-        {name: 'v2', prerelease: true},
+        {name: 'v2-alpha', prerelease: true},
+        {name: 'v2-alpha-publish-releases', prerelease: true},
```

Comment:
do we need this? is that not already covered through the v2-alpha?

Reply:
also looks too much for me

Reply:
ah this is the current branch to test the changes